### ISSUE TITLE
Font Active Event Not Firing With Multiple Iframes

### ISSUE THREAD

**Issue by cmagnollay** [typekit/webfontloader#369]
I have set up a fiddle to demonstrate this issue:

https://jsfiddle.net/kadogdcq/2/

**What I expect**:

Two console messages informing me that the fonts are active in both iframe1 and iframe2. The same content and styling in both iframes.

**What I'm seeing**:

The active event is not fired for the second iframe, despite this set of classes present on the <html> tag: 

"wf-droidsans-n4-active wf-active"

**Workaround**:

For whatever reason setting a timeout to throttle the calls prevents this issue. For for example putting 800ms between calling WebFont.load fixes this issue.

Am I missing something here? I'm happy to provide further information if needed.

**Comment by cmBarsotti** [comment]
I'm having this exact problem but only in IE (!!!)

**Comment by lucksp** [comment]
Have you tried the `context: frames` [configuration](https://github.com/typekit/webfontloader#iframes)?

**Comment by cmagnollay** [comment]
@lucksp I am using the context configuration in the example I provided. Are you talking about something different? Could you explain how you would like me to use it differently please?

**Comment by lucksp** [comment]
You're right, i misread when glancing, but take a look at some of the changes for the context definition:
https://jsfiddle.net/kadogdcq/5/

Based on the concept of the [PR#71](https://github.com/typekit/webfontloader/pull/71/files) that they used, the iFrame context is based on a "name" attribute rather than an ID or Class name.  Perhaps I am misreading that PR?

Anyway, also some tweaks to getting the content of the iframe into the body rather than the html.

**Comment by pikooli** [comment]
Yeah you have to use the name attr for the iframe.
This example help me understand how it work.
https://github.com/typekit/webfontloader/blob/master/lib/webfontloader/demo/public/google-iframe.html
